gitbook-docs/market/economics.md

```diff
@@ -1,124 +1,183 @@
+This approach allows the proof requester, whose proof will be reused, to recoup proof costs
+in the future and maybe even earn something on it.
+Similarly, a proof producer can sell their proof more than once and receive a commission
+from each such additional sale.
 
-#### Proof Requester
+## Registration
+### Proof requester
 
-There is no process of registration for proof buyers. Any =nil;Cluster account is able to send a bid to the marketplace. The only requirement is that the user's account must have enough funds to pay for the order.
+There is no process for proof requesters registration.
+Any `=nil;` cluster account can send a request to the marketplace.
+The only requirement is that the user's account must have enough funds to pay for the order.
 
-#### Proof Producer
+### Proof producer
 
-Account has to register as a Proof Producer to be able to set asks to the marketplace. The process includes locking funds (_deposit_). The number of locked funds reflects the number of orders that the proof producer can process at the moment.
+Account has to register as a Proof Producer to put proposals on the marketplace.
+The process includes locking funds (_deposit_).
+The number of locked funds reflects the number of orders that the proof producer can process at the moment.
 
-Let _X_ is the number of locked funds by the proof producer. Then they can process bids on the sum of _X/pf_ at once. This restriction is related to the penalties policy described in the next sections.
+Let _X_ be the number of locked funds by the proof producer.
+Then they can process requests on the sum of _X/pf_ at once.
+This restriction is related to the penalties policy described in the following sections.
 
-### Funds transferring & commissions
+## Funds transferring & commissions
 
-When placing a bid, the customer sets the cost that he is willing to pay for the successfully submitted proof for the bid. If the proof for the bid is provided on time, the amount indicated in the ask cost is deducted from the customer's account and credited to the account of the proof producer.
+When placing a request, the proof requester sets the cost they are willing to pay for
+the successfully submitted proof.
+If the proof for the request is provided on time, the price indicated in the proposal cost
+is deducted from the proof requester's account and credited to the account of the proof producer.
 
-_c_% of the bid cost goes as the commission but not less than _mc_ tokens. It is split between two parties
+_c_% of the request cost goes as the commission, but not less than _mc_ tokens.
+It's split between two parties:
 
 * _app\_dev_ part goes to the Proof Market development team;
-* _cir\_dev_ part goes to the Circuit Developer of the ordered statement.
+* _cir\_dev_ part goes to the circuit developer of the ordered statement.
 
-#### Funds withdraw
+### Funds withdrawal
 
-**From the proof requester's perspective:**
+#### From the proof requester's perspective
```

Comment:
I like that you converted these wannabe-headings into real ones.

The previous heading is `##`, so this should be `###`, same for the next one.

Reply:
Funds transferring & commissions was a 2nd level heading, not the previous one
Economics -> Funds transferring & commissions -> Funds withdrawal -> From the proof requester's perspective. Looks like a 4th level heading to me

Reply:
oh, didn't notice it :)